### PR TITLE
EES page tidy up

### LINKED ISSUE #69: Improve description of comma delimited structure
**Is your feature request related to a problem? Please describe.**
Comment on https://github.com/dfe-analytical-services/dfe-published-data-qa/pull/129 that the guidance we point to might not help users who aren't familiar with the terminology to know what to do.

**Describe the solution you'd like**
An improved description of comma delimited structure and how delimited structures work / how to save files in that format.

**Describe alternatives you've considered**
None

**Additional context**
Related section beneath around saving as UTF-8 format. If either this header or that header changes we'll need to make sure to update the links in the data screener as we currently link to both sections a lot

### LINKED ISSUE #98: Minor fixes to EES page required
**Is your feature request related to a problem? Please describe.**

Several (perhaps an understatement) fixes required for this page: https://dfe-analytical-services.github.io/analysts-guide/statistics-production/ees.html

---

- [x] Think this mailbox needs changing to EES mailbox:
image

---

- [x] "Accessing EES" is a level 1 header

---

- [x] Should this be EES team instead of SDT? Happy with either:
image

---

- [x] Inconsistent capitalisation / naming of mailbox links
image
image

---

- [x] No mailbox link
image

---

- [x] Typo in accessibility
image

---

- [x] Typo in "once"
image

---

- [x] Broken image reference
image

---

- [x] Code block formatting needs fixing
image

---

- [x] Lots of bold in this section
image

---

- [x] Another broken image
image

---

- [x] Another missing mailbox link
image

---

- [x] Wrong its
image

---

- [x] Another broken image
image

---

- [x] Stats badging names need changing?
image

---

### PR DESCRIPTION
**Pull request by cjrace**
## Overview of changes

Addressing outstanding issues referenced in #98 and #69.

## Why are these changes being made?

General tidy up of the page, updating mailboxes after team split, removing h1s.

On the data page, added a bit more detail around CSVs and standardised the way we reference them to meet style guides.

## Detailed description of changes

Can be found in #98.

## Issue ticket number/s and link

Resolves #98. Resolves #69.

## Checklist before requesting a review
- [x] I have checked the contributing guidelines
- [x] I have checked for and linked any relevant issues that this may resolve
- [x] I have checked that these changes build locally
- [x] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

File: statistics-production/ees.qmd [hunk 1]
```diff
@@ -281,9 +296,11 @@ The process for adding external users as collaborating analysts is similar to ad
 
 ---
 
-### What to include in an external access request
+#### What to include in an external access request
 
-If you have external users you'd like to request access for, please send the following details to the [Explore Statistics mailbox](mailto:explore.statistics@education.gov.uk), **at least two weeks** in advance of requiring access, but ideally as early as possible. Due to the dependency on DfE's digital security, we cannot guarantee access or how long it may take.
+---
+
+If you have external users you'd like to request access for, please send the following details to the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk), **at least two weeks** in advance of requiring access, but ideally as early as possible. Due to the dependency on DfE's digital security, we cannot guarantee access or how long it may take.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:303]: Could the two week thing be in a callout?

File: statistics-production/ees.qmd [hunk 2]
```diff
@@ -433,87 +450,88 @@ This then becomes the default order for this filter or indicator and will apply
...
+---
 
-One you've chosen and uploaded your replacement file it will need to go through the usual import process before it can check if retaining existing data blocks and footnotes will be possible.
+### Public data guidance
 
-![](../images/replace-data-step2.PNG)
+---
 
-Once the upload is finished a report will appear which highlights whether existing data blocks and footnotes can be replaced successfully. If you want to keep any data blocks and footnotes you've built you will need to make sure that your replacement data file still contains the information (indicators, filters, geographic_levels and time_periods) that was used to create them.
+Within the 'Data and files' page, you can also create your public data guidance. This replaces the information that would have previously been uploaded as a pdf on GOV.UK and is designed to help users understand and use the data they download from your release. See the [Permanent and fixed period exclusions data guidance](https://explore-education-statistics.service.gov.uk/find-statistics/permanent-and-fixed-period-exclusions-in-england/meta-guidance) for an example.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:516]: PDF should be in caps

File: statistics-production/ees.qmd [hunk 3]
```diff
@@ -433,87 +450,88 @@ This then becomes the default order for this filter or indicator and will apply
...
+![](../images/admin-public-meta-start.PNG)
 
-![](../images/replace-data-fail.PNG)
+A list of variables in each file with an associated label (taken from metadata uploads) and associated footnotes will also be displayed for each file.
+
+![](../images/admin-public-meta-detail.PNG)
 
 <div class="alert alert-dismissible alert-danger">
-Remember to double check any data blocks or footnotes that were recreated by the platform before publishing your release.
+Releases cannot be published without a completed data guidance. If not filled in an error will be flagged during sign off.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:527]: "a completed data guidance" sounds odd - is there a word missing?

File: statistics-production/ees.qmd [hunk 4]
```diff
@@ -893,8 +912,23 @@ The above code should produce something along the lines of the following plots:
 
 ---
 
-## Content
+#### Embedding R-Shiny blocks
+
+---
+
+If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R-Shiny produced chart.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:919]: Is R-Shiny the correct way to write it? We have other variants elsewhere (e.g. "R Shiny", "RShiny")

File: statistics-production/ees.qmd [hunk 5]
```diff
@@ -1240,7 +1264,12 @@ Once the release has been marked as approved, go to the  ‘Pre-release access
 
 ![](../images/EES-invite-pra.PNG)
```
Comment (jen-machin) [inline statistics-production/ees.qmd:1265]: this .PNG is caps - will it cause issues?

File: statistics-production/ud.qmd [hunk 6]
```diff
@@ -277,10 +273,14 @@ If your current processes produce wide data that you need to switch to a tidy st
 These standards give you the power to format the data in a way that best meets the needs of the users. There are only a handful of formatting standards to follow to ensure best practice and consistency across all of our data.
 
 <div class="alert alert-dismissible alert-danger">
-Data files must in comma separated values (.csv) format, and use UTF-8 encoding. You can specify this when saving the file in Excel, or exporting from elsewhere.
+Data files must in comma separated values (CSV) format, and use UTF-8 encoding. You can specify this when saving the file in Excel, or exporting from elsewhere.
 </div>
 
-If you need to use commas within a cell, then you must add a text delimiter such as quotes to your file to define each cell - this is often done automatically for you, though if you're unsure then you can open up your csv file in a text editor such as notepad to check.
+If you need to use commas within a cell, then you must add a text delimiter such as quotes to your file to define each cell - this is often done automatically for you, though if you're unsure then you can open up your CSV file in a text editor such as notepad to check.
```
Comment (jen-machin) [inline statistics-production/ud.qmd:279]: Notepad should probably have a capital N

File: statistics-production/ees.qmd (hunk 5 above)
Comment (cjrace) [reply statistics-production/ees.qmd:1265]: Nope - it just needs to match what is in the folder

File: statistics-production/ud.qmd [hunk 7]
```diff
@@ -277,10 +273,14 @@ If your current processes produce wide data that you need to switch to a tidy st
...
+
+Comma separated values simply means that if look at the raw file, each column is separated by a comma, other forms of delimination like this also exist, such as TSV files that are 'tab' separated. Next time you have a minute, try opening up your CSV data files in notepad, to see how they look behind the scenes!
+
+Saving files in CSV format should be a standard from any analytical tool, though sometimes you need to watch out for how they are encoding special characters, for more information see the section on [how to export data with UTF8 encoding](#how-to-export-data-with-utf-8-encoding).
```
[the remark this answers is not in the document]
Comment (cjrace) [reply statistics-production/ud.qmd:283]: Yeah missed a word there, updated!

File: statistics-production/ud.qmd [hunk 8]
```diff
@@ -277,10 +273,14 @@ If your current processes produce wide data that you need to switch to a tidy st
...
-Data files must in comma separated values (.csv) format, and use UTF-8 encoding. You can specify this when saving the file in Excel, or exporting from elsewhere.
+Data files must in comma separated values (CSV) format, and use UTF-8 encoding. You can specify this when saving the file in Excel, or exporting from elsewhere.
 </div>
 
-If you need to use commas within a cell, then you must add a text delimiter such as quotes to your file to define each cell - this is often done automatically for you, though if you're unsure then you can open up your csv file in a text editor such as notepad to check.
+If you need to use commas within a cell, then you must add a text delimiter such as quotes to your file to define each cell - this is often done automatically for you, though if you're unsure then you can open up your CSV file in a text editor such as notepad to check.
+
+Comma separated values simply means that if look at the raw file, each column is separated by a comma, other forms of delimination like this also exist, such as TSV files that are 'tab' separated. Next time you have a minute, try opening up your CSV data files in notepad, to see how they look behind the scenes!
```
[the remark this answers is not in the document]
Comment (cjrace) [reply statistics-production/ud.qmd]: Good catch!

File: statistics-production/ud.qmd (hunk 6 above)
Comment (cjrace) [reply statistics-production/ud.qmd]: Good catch!

File: statistics-production/ees.qmd (hunk 4 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Nope, should be R Shiny! Fixed, and did a sweep of other ones too

File: statistics-production/ees.qmd (hunk 3 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Good spot, added

File: statistics-production/ees.qmd (hunk 2 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Fixed

File: statistics-production/ees.qmd (hunk 1 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Yep, added!

File: statistics-production/ees.qmd [hunk 9]
```diff
@@ -912,15 +918,15 @@ The above code should produce something along the lines of the following plots:
...
-#### Embedding R-Shiny blocks
+#### Embedding R Shiny blocks
 
 ---
 
-If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R-Shiny produced chart.
+If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R Shiny produced chart.
 
-To embed a shiny chart, you'll first need to create an R-Shiny application containing the chart using the DfE tiny-shiny template and get it hosted on our DfE GitHub and ShinyApps accounts. More details on both of these are available in the tiny-shiny section of this guidance.
+To embed a shiny chart, you'll first need to create an R Shiny application containing the chart using the DfE tiny-shiny template and get it hosted on our DfE GitHub and ShinyApps accounts. More details on both of these are available in the tiny-shiny section of this guidance.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:927]: Think the first Shiny in this sentence should have big S? Sorry, missed it on the first go

File: statistics-production/ees.qmd [hunk 10]
```diff
@@ -74,11 +75,13 @@ The pre-production environment is not suitable for unpublished data. Unpublished
 
 ---
 
-### Getting access to admin
+#### Getting access to admin
+
+---
 
-Access to both the production and pre-production admin services is limited to DfE AD accounts only and users have to have been invited to the service by either the Statistics Development Team (for full access) or an existing user (for pre-release access). Invites to the service are sent out via email using gov.uk Notify.
+Access to both the production and pre-production admin services is limited to DfE AD accounts only and users have to have been invited to the service by either the explore education statistics platforms team (for full access) or an existing user (for pre-release access). Invites to the service are sent out via email using gov.uk Notify.
```
Comment (jen-machin) [inline statistics-production/ees.qmd:82]: "gov.uk Notify" - should be GOV.UK

File: statistics-production/ees.qmd [hunk 11]
```diff
@@ -893,8 +918,23 @@ The above code should produce something along the lines of the following plots:
...
-## Content
+#### Embedding R Shiny blocks
+
+---
+
+If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R Shiny produced chart.
+
+To embed a shiny chart, you'll first need to create an R Shiny application containing the chart using the DfE tiny-shiny template and get it hosted on our DfE GitHub and ShinyApps accounts. More details on both of these are available in the tiny-shiny section of this guidance.
+
+Once you've got the R Shiny app set up and hosted, you can embed it using the **Add embed block** button in the content area. This will give you the option to enter a URL, where you can enter the URL of the shiny app. Valid URLs to use in the embed block dialogue box are limited to only those on the DfE ShinyApps server (and the internal rsconnect when needed for draft publications).
```
Comment (jen-machin) [inline statistics-production/ees.qmd:929]: should this be POSIT connect?

File: statistics-production/embedded-charts.qmd [hunk 12]
```diff
@@ -2,15 +2,15 @@
...
+<p class="text-muted">Guidance for creating and embedding R Shiny visualisations in EES publications</p>
 
 ---
 
 # Introduction
 
 ---
 
-EES now offers the option to add an embedded block, which allows analysts to present custom R created charts within their publications. This necessitates analysts creating an R-Shiny based app on GitHub containing the required chart that can then be hosted on a shiny server (currently using the DfE's account on shinyapps.io).
+EES now offers the option to add an embedded block, which allows analysts to present custom R created charts within their publications. This necessitates analysts creating an R Shiny based app on GitHub containing the required chart that can then be hosted on a shiny server (currently using the DfE's account on shinyapps.io).
```
Comment (jen-machin) [inline statistics-production/embedded-charts.qmd:13]: Think this shiny needs a big S also

File: statistics-production/pub.qmd [hunk 13]
```diff
@@ -66,7 +66,7 @@ All national, official and ad-hoc statistics should be published on EES. This se
 ## Publication checklist
 
 
-Before releasing statistics for the first time you may want to discuss the new process with key stakeholders and/or pre-release users to make them aware of the new service. You should also inform the [Explore Statistics Mailbox](mailto:explore.statistics@education.gov.uk) and [HoP Office](mailto:Hop.STATISTICS@education.gov.uk) teams.
+Before releasing statistics for the first time you may want to discuss the new process with key stakeholders and/or pre-release users to make them aware of the new service. You should also inform the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk) and [Statistics HoP Office](mailto:Hop.STATISTICS@education.gov.uk).
 
 Before you start creating a release in the platform you should have:
```
Comment (jen-machin) [inline statistics-production/pub.qmd:72]: Line 73 has a gov.uk instead of GOV.UK

Comment (cjrace) [reply statistics-production/pub.qmd:72]: Fixed

File: statistics-production/ees.qmd [hunk 14]
```diff
@@ -1216,7 +1241,12 @@ Prior to each release going live the production team are also able to grant pre-
...
 This preview is only accessible for the 24 hours before the publication date, although the emails may go out to users before then. We expect teams will continue to send an email at 09:30 on pre-release day, including any additional briefing and the link to the pre-release area.
 
+---
+
 ### Granting pre-release access
+
+---
+
 <div class="alert alert-dismissible alert-info">
 Jobshare emails and shared mailboxes should not be sent invites for pre-release. The personal emails for the specific individuals should be used instead, as jobshare emails/shared mailboxes do not have active accounts with DfE to access EES.
```
[the remark this answers is not in the document]
Comment (cjrace) [reply statistics-production/ees.qmd]: Yeah 100%, better for accessibility so it doesn't treat it as a single word, good spot!

File: statistics-production/ees.qmd (hunk 10 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Fixed

File: statistics-production/ees.qmd (hunk 9 above)
Comment (cjrace) [reply statistics-production/ees.qmd]: Sorted

File: statistics-production/embedded-charts.qmd (hunk 12 above)
Comment (jen-machin) [reply statistics-production/embedded-charts.qmd]: Literally just spotted that level 1 Introduction header above this on line 9 🤣 worth fixing while we're here

Comment (jen-machin) [reply statistics-production/embedded-charts.qmd]: Big S is fixed though

File: statistics-production/pub.qmd [hunk 15]
```diff
@@ -66,11 +66,11 @@ All national, official and ad-hoc statistics should be published on EES. This se
...
-Before releasing statistics for the first time you may want to discuss the new process with key stakeholders and/or pre-release users to make them aware of the new service. You should also inform the [Explore Statistics Mailbox](mailto:explore.statistics@education.gov.uk) and [HoP Office](mailto:Hop.STATISTICS@education.gov.uk) teams.
+Before releasing statistics for the first time you may want to discuss the new process with key stakeholders and / or pre-release users to make them aware of the new service. You should also inform the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk) and [Statistics HoP Office](mailto:Hop.STATISTICS@education.gov.uk).
 
 Before you start creating a release in the platform you should have: 
 
-* Announced the upcoming release via gov.uk
+* Announced the upcoming release via GOV.UK
 * Sent metadata form to HoP
 * Contacted the BAU team so we can support you with your first release
 * Produced your tidy csv data files with appropriate disclosure control
```
Comment (jen-machin) [inline statistics-production/pub.qmd:76]: Oh noooo small csv 😞 lines 76 and 77

File: statistics-production/embedded-charts.qmd (hunk 12 above)
Comment (cjrace) [reply statistics-production/embedded-charts.qmd]: I think every page we haven't recently touched has level 1 headers and the whole rest of the headers on the page also need adjusting to fit... 

I've done this one but I think for anymore we should wait until one big sweep to do the rest as it always requires altering every header on the page, which can take time for bigger pages.

File: statistics-production/pub.qmd (hunk 15 above)
Comment (cjrace) [reply statistics-production/pub.qmd]: Fixed, made them BIG

**Pull request merged**